pycbc/waveform/nr_waveform.py

```diff
@@ -143,7 +138,10 @@ def get_param(value):
 
     # Restrict start time if needed
     print template_params
-    est_start_time = seobnrrom_length_in_time(**template_params)
+    try:
+        est_start_time = seobnrrom_length_in_time(**template_params)
+    except:
+        est_start_time = -time_start_s
```

Comment:
This is a hack. Basically to be able to line waveforms up at the *start* frequency I need to generate the whole thing. Unfortunately seobnrrom_length_in_time will normally give an out of valid domain error if I try to do this!